src/modules/awake/Awake/Program.cs

```diff
@@ -180,7 +180,7 @@ private static void HandleCommandLineArguments(bool usePtConfig, bool displayOn,
                         .Select(e => e.EventArgs)
                         .Subscribe(HandleAwakeConfigChange);
 
-                    TrayHelper.SetTray(AppName, new AwakeSettings());
+                    TrayHelper.SetTray(FullAppName, new AwakeSettings());
```

Comment:
I still get "Awake" in the tray when testing this PR.
There might be more places where the `AppName` -> `FullAppName` change needs to happen.